addon/components/async-button.js

```diff
@@ -12,10 +12,17 @@ export default Ember.Component.extend({
   reset: false,
   classNames: ['async-button'],
   classNameBindings: ['textState'],
-  attributeBindings: ['disabled', 'type', '_href:href'],
+  attributeBindings: ['disableWhen', 'disabled', 'type', '_href:href', 'tabindex'],
 
   type: 'submit',
-  disabled: Ember.computed.equal('textState','pending'),
+  disabled: Ember.computed('textState', 'disableWhen', function() {
+      var textState = this.get('textState');
```

Comment:
it is part of our styleguides https://github.com/dockyard/styleguides/blob/master/javascript.md#variables

The `var`s that are currently there are artifacts from before we implemented the styleguide. Any future changes to our libraries should adhere